agent/scripts/fetch-issues.js

```
@@ -204,15 +204,15 @@ async function fetchIssues() {
   // ─────────────────────────────────────────────
   // Ensure data directory exists
   // ─────────────────────────────────────────────
-  if (!fs.existsSync('./data')) {
-    fs.mkdirSync('./data', { recursive: true });
+  if (!fs.existsSync(path.resolve(__dirname, '../../data'))) {
```

ESLint

[error] 'path' is not defined.

(no-undef)